docs/git-comparison.md

```diff
@@ -211,6 +211,11 @@ parent.
       <td><code>git reset --hard</code> (same as abandoning a change since Git
           has no concept of a "change")</td>
     </tr>
+    <tr>
+      <td>Abandon the current change, but keep its diff in the working copy</td>
```

Comment:
It's the *parent* change we abandon here, so I would say "Abandon the parent change" here. I hope that's not too confusing.

Btw, Mercurial, which has no automatic snapshotting of the working copy as a commit, also calls it "the parent of the working copy" :) I remember thinking that term was confusing, but it now makes a lot of sense to me.

Reply:
i think "parent of the working copy" is great phrasing, since we want this to also apply for git :) i'll use that.